packages/xrpl/tools/createValidateTests.js

```diff
@@ -0,0 +1,197 @@
+const fs = require('fs')
```

Comment:
This is mostly the same as the old `createValidateTests` file (I didn't even change very much), but I guess git doesn't think they're the same file anymore.